shell/pages/c/_cluster/explorer/index.vue

```diff
@@ -138,6 +139,26 @@ export default {
     this.$store.dispatch('cluster/forgetType', MANAGEMENT.NODE);
   },
 
+  watch: {
+    // logic to be removed once kube version is 1.25 or greater
+    // we need to hook up this API call to a watcher because the page logic is based on a getter
+    // as a temporary solution, this seems reasonable, so that we don't disrupt the optimal loading times of the page
+    currentCluster: {
+      async handler(neu, old) {
```

Comment:
Is this code expected to execute if you refresh the page on cluster explorer? I'm not sure if the watcher would be triggered in that case, but the banner didn't show for me regardless of whether I refreshed or navigated from a different cluster.

Reply:
Yes, as the watcher has the `immediate: true ` prop + the conditions on the watch should mean that it only runs once (not triggering multiple API calls to fetch policies)